apps/condo/domains/organization/utils/tin.utils.spec.js

```diff
@@ -2,15 +2,23 @@ const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
 const { isValidTin } = require('@condo/domains/organization/utils/tin.utils')
 
 const VALID_RU_TIN_10 = '1654019570'
+const VALID_RU_TIN_10_1 = '6311095616'
+const VALID_RU_TIN_10_SPC = '    1654019570    '
```

Comment:
Please add more cases…like tabulations, non-breaking spaces and so

Reply:
done